src/test/groovy/com/avioconsulting/mule/linter/rule/pom/MuleRuntimeVersionRuleTest.groovy

```diff
@@ -3,17 +3,25 @@ package com.avioconsulting.mule.linter.rule.pom
 import com.avioconsulting.mule.linter.model.Application
 import com.avioconsulting.mule.linter.model.Rule
 import com.avioconsulting.mule.linter.model.RuleViolation
+import com.avioconsulting.mule.linter.TestApplication
 import spock.lang.Specification
 
 class MuleRuntimeVersionRuleTest extends Specification {
 
-    private static final String APP_NAME = 'SampleMuleApp'
+    private final TestApplication testApp = new TestApplication()
```

Comment:
If this is what is creating the directory, it should probably be in setup to match the remove in the cleanup.  Or just move the  actual creation of the TestApplication tmp dir to a method instead of the constructor, and call that from setup.  We just want to make sure we have matching "create app" and "remove app" functionality in setup and cleanup.